kolibri/plugins/coach/assets/src/app.js

```diff
@@ -35,22 +34,19 @@ class CoachToolsModule extends KolibriModule {
           },
         },
         {
-          name: PageNames.RECENT_CHANNEL_SELECT,
-          path: '/:classID',
-          handler: (toRoute, fromRoute) => {
-            recentActions.showChannelSelect(store, toRoute.params.classID);
-          },
-        },
-        {
           name: PageNames.RECENT,
-          path: '/:classID/:channelID',
+          path: '/:class_id/recent/:channel_id?',
```

Comment:
would like to change everything to `camelCase` eventually, but at the moment coach URLs tend to use underscores because they mapped to API parameters that way.

down the line, I agree we should change.